modules/odc-viewing-application-topology.adoc

```diff
@@ -5,29 +5,24 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
-
-
-The status or phase of the Pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
-
-After you create an application and an image is deployed, the status is shown as *Pending*. After the application is built, it is displayed as *Running*.
-
-.Application topology
-image::odc_application_topology.png[]
-
-The application resource name is appended with indicators for the different types of resource objects as follows:
-
-* *CJ*: `CronJob`
-* *D*: `Deployment`
-* *DC*: `DeploymentConfig`
-* *DS*: `DaemonSet`
-* *J*: `Job`
-* *P*: `Pod`
-* *SS*: `StatefulSet`
-* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application
-+
-[NOTE]
-====
-Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology* view. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology* view.
-====
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```

Comment:
Again, I see several different version of bolding just Topology, or both Topology and view, and in the preview module, View was capitalized when they were both bolded. Just need some more consistency here.

Reply:
Hi Andrea, adding some context: We realized this was going to cause confusion so we have initiated a conversation with the dev and UX teams. On the left navigation panel one of the options we have is Topology, clicking on this shows the Topology view for apps. This view has 2 options List view and Topology view. So we are in a weird situation where we refer to **Topology** view and **Topology view** as per the context. As per the devs, *Topology view* should be *Graph view* and we will update it as soon as we see this change in.
![Screenshot from 2021-01-19 14-18-05](https://user-images.githubusercontent.com/16647625/105010619-f72a3c80-5a61-11eb-804f-fa6b2348c983.png)

Reply:
Thanks for the explanation! I think my main concern this time was that "View" was also capitalized. But it looks like that was taken care of in the new PR, so we are good to go!